### DISCUSSION TITLE
Who will make a deal in data economy? And with whom?

### DISCUSSION THREAD

**Discussion by tktkowash** [JasmyCoin/JasmyCoin#21] [General]
Who will make a deal in data economy? And with whom?

**Comment by harayamada** [comment]
ex: Assuming Panasonic products below .

This is the refrigerator with "Stock Manager" that allows you to manage remaining quantity information based on the weight of food ingredients with an app. By registering eggs and other foodstuffs that you want to manage in the "Kitchen Pocket" of the app, and placing them on the weight detection plate, the IoT function automatically updates the remaining quantity. You can check it even when you are out shopping. Inventory information can be shared with family members, allowing them to shop according to their usage conditions. With the notification function of the remaining amount and expiration date set, food loss due to forgotten use and menu changes due to forgotten purchases can be reduced.
